templates/docs/patterns/icons.md

```diff
@@ -68,7 +68,7 @@ Our icons have two predefined color styles: light and dark. The light variant is
         <p><i class="p-icon--external-link" style="margin-right: 1rem;"></i>p-icon--external-link</p>
       </div>
       <div class="p-card col-3 u-vertically-center">
-        <p><i class="p-icon--contextual-menu" style="margin-right: 1rem;"></i>p-icon--contextual-menu</p>
+        <p><i class="p-icon--chevron-down" style="margin-right: 1rem;"></i>p-icon--chevron-down</p>
```

Comment:
Please also add the `chevron-up` to the list.

Reply:
There is also a "deprecated" section below these icons. Would be good to mention `contextual-menu` there.

Reply:
Good points - have addressed all of these.